Makefile

```diff
@@ -8,24 +8,27 @@ clean:
 quality:
 	tox -e quality
 
-requirements: ## install development environment requirements
+## install development environment requirements
+requirements:
```

Comment:
The comment needs to stay on the same line as the target for `make help` to work correctly.